app/javascript/components/DownloadButton.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faDownload } from '@fortawesome/free-solid-svg-icons';
 import Modal from 'react-bootstrap/lib/Modal';
+import Tooltip from 'react-bootstrap/lib/Tooltip';
```

Comment:
is this import directly used in this file?  Or just needed so the js for data-toggle='tooltip' is loaded?  If the latter, this might be better imported in application.js since we will probably use it in other places too.

Reply:
This will be used directly used in this file -- but later, when refining `onClipboardCopySuccess` to briefly display a tooltip.  I've also added a comment to this effect in 68fc531.

In other words, this will get directly used when the TODO (referencing SCP-2167) is done.